v2/card_pools/standard.json

```diff
@@ -2,6 +2,7 @@
   {
     "code": "pre_rotation",
     "name": "Pre Rotation",
+    "format_id": "standard",
```

Comment:
would you please add a test that ensures that all the format_ids in these files match the file names?

Reply:
Done!